### DISCUSSION TITLE
GitHub Student Developer Pack Approved But Not Able to Create Microsoft Azure Account.

### DISCUSSION THREAD

**Discussion by Samar029** [community/community#135932] [GitHub Education]
### Hi everyone,

My GitHub Student Developer Pack is approved and i want to sign up Microsoft Azure Student account but when i am going to create it gives an error "We could not confirm your status as a student. Click [here](https://go.microsoft.com/fwLink/p/?LinkID=2136428&clcid=0x409) to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students."
Please i need solution to access Microsoft Azure Student Account

**Comment by davevad93** [comment]
Hi @Samar029 ,

This is a known problem as seen in these discussions: #131336 , #135739

You have two options to solve it: 

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

2.  Create a support ticket directly on [MS Azure](https://azure.microsoft.com/en-us/support/create-ticket).

The user in discussion #131336 solved it by contacting Azure.
